uipyqt.py

```diff
@@ -0,0 +1,288 @@
+import sys
+from PyQt5.QtWidgets import *
```

Comment:
SonarLint: Import only needed names or import the module and then use its members.